packages/design-tokens/src/figma/tokens.json

```diff
@@ -514,18 +514,6 @@
       },
       "type": "typography"
     },
-    "L-Heading-SM-Regular": {
-      "value": {
-        "fontWeight": "$fontWeights.regular",
-        "lineHeight": "$lineHeights.4",
-        "fontSize": "$fontSize.5",
-        "letterSpacing": "$letterSpacing.0",
-        "paragraphSpacing": "$paragraphSpacing.0",
-        "textCase": "$textCase.none",
-        "textDecoration": "$textDecoration.none"
-      },
-      "type": "typography"
-    },
```

Comment:
Removing deprecated tokens from Figma json. Ensured it's not used in the codebase

<img width="498" alt="Screenshot 2025-05-25 at 2 22 09 PM" src="https://github.com/user-attachments/assets/df6d1765-4cb6-4e70-aee3-cf6df99bc58e" />